### PR TITLE
doc/user: remove preview label from "cc" sizes

### PR DESCRIPTION
**Pull request by benesch**
The "cc" sizes have proved to be stable during their preview period. We're ready to make them available by default, as the first step towards making them GA (MaterializeInc/cloud#8668).

### Motivation

   * This PR updates documentation.

### Checklist

- [x] This PR has adequate test coverage / QA involvement has been duly considered.
- [x] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [x] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [x] If this PR will require changes to cloud orchestration or tests, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [x] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
  - n/a

### CODE REVIEW

File: doc/user/layouts/shortcodes/postgres-direct/right-size-the-cluster.html [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
-   command to downsize the cluster to `xsmall`:
+   command to downsize the cluster to `100cc`:
 
     ```sql
-    ALTER CLUSTER ingest_postgres SET (SIZE 'xsmall');
+    ALTER CLUSTER ingest_postgres SET (SIZE '100cc');
     ```
 
-    Behind the scenes, this command adds a new `xsmall` replica and removes the
+    Behind the scenes, this command adds a new `100cc` replica and removes the
```
Comment (sjwiesman) [inline doc/user/layouts/shortcodes/postgres-direct/right-size-the-cluster.html:13]: There’s a reference to small on the line below this one. And an xsmall reference a the top of the page

**Review by JLDLaughlin — APPROVED** [review]
🚗

File: doc/user/layouts/shortcodes/postgres-direct/right-size-the-cluster.html (hunk 1 above)
Comment (benesch) [reply doc/user/layouts/shortcodes/postgres-direct/right-size-the-cluster.html:13]: Thanks, fixed!

**Comment by benesch** [comment]
Planning to merge this after we meet on Monday afternoon.

**Pull request merged**